aiperf/common/config/input_config.py

```diff
@@ -218,6 +218,24 @@ def validate_dataset_type(self) -> Self:
         ),
     ] = InputDefaults.RANDOM_SEED
 
+    goodput: Annotated[
+        Any | None,
```

Comment:
I had to do some somewhat hacky logic to get this to work for headers and extra-inputs, due to them needing to able to be specified more than once. this approach using Any should be fine as long as we don't want to let users specify it as `--goodput "itl:10" --goodput "request_latency:1000"`, etc.